state_mixin.py

```diff
@@ -14,6 +15,9 @@ class StateMixin(object):
     state_expr = ExpressionProperty(title='State Expression', default='{{$state}}')
     backup_interval = TimeDeltaProperty(title='Backup Interval',
                                         default={'seconds': 600})
+    load_from_persistence = BoolProperty(title="Load from persistence?", default=True,
```

Comment:
If this is in the `StateMixin` parent class, why does it need to get added to each block too?